tests/form.test.ts

```diff
@@ -17,16 +17,15 @@ describe("form", () => {
     let word
     let map
     let t
+    let f: Form
```

Comment:
why make it a property and share it between tests?

Reply:
Why not? It's testing forms only so every test initializes it anyway. Also, this way it is much easier to handle `onKey` events (especially after the refactoring done to the form) - see near the end of `beforeEach` where we set the terminal's `onKey` to be directed to the form's one.

Reply:
Because it's better to keep the unit tests as isolated as possible.  One of the most annoying things (and hard to find) is  when a test fails not because of the previous test. As a design pattern it's better to block any way data can creep from one test to the next.  For example, calling `terminal.onKey()` adds a key handler without removing the previous one.

If it complicates things we can make an exception, but if it's just a question of one or two more lines at the beginning of each test I prefer to keep tests as separate as possible. 

(BTW - It's different in the E2E tests where each test is a step and they have to run one after the other so they must share data).